Error running cx in blockchain broadcast mode

**Describe the bug**
Following the [CX Chains Tutorial Wiki](https://github.com/skycoin/cx/wiki/CX-Chains-Tutorial) I get the following error when I attempt to run CX in broadcast mode (`--braodcast`).

```bash
cx --broadcast --heap-initial 100 --stack-size 100 examples/blockchain/counter-txn.cx
Blockchain counter: 1
panic: invalid character 'B' after top-level value

goroutine 1 [running, locked to thread]:
main.main()
	/go/src/github.com/skycoin/cx/cxgo/main.go:857 +0x549a
```

**To Reproduce**
Steps to reproduce the behavior:
1. Follow the steps outlined in the [CX Chains Wiki TL;DR section](https://github.com/skycoin/cx/wiki/CX-Chains-Tutorial)
2. Once you have both a `publisher` and a `peer` node running (in two seperate terminal sessions) run the test broadcast using the following command:
```bash
cx --transaction --heap-initial 100 --stack-size 100 examples/blockchain/counter-txn.cx
```

This succeeds.

3. Next try to broadcast the transaction using the following command:
```bash
cx --broadcast --heap-initial 100 --stack-size 100 examples/blockchain/counter-txn.cx
```

This fails with the following error:
```bash
Blockchain counter: 1
panic: invalid character 'B' after top-level value

goroutine 1 [running, locked to thread]:
main.main()
	/go/src/github.com/skycoin/cx/cxgo/main.go:857 +0x549a
```

The error points to this block of code:
https://github.com/skycoin/cx/blob/4ba0d2d117e625622cc1a4224932ec4648602a49/cxgo/main.go#L855-L858

Debugging somewhat, I appear to be getting a `400 Bad Request` returned from this block of code:
https://github.com/skycoin/cx/blob/4ba0d2d117e625622cc1a4224932ec4648602a49/cxgo/main.go#L843-L847


**Expected behavior**
The transaction should be broadcast successfully onto the cxcoin blockchain.
Possibly some information is missing in the guide, or something is incorrectly configured.

**Screenshots**
NA. Refer to error information provided.

**Desktop (please complete the following information):**
 - OS: MacOS 10.14.4 (Mojave)
 - Browser: NA
 - CX Version: 0.6.2 - develop branch with blockchain integration. https://github.com/skycoin/cx/commit/4ba0d2d117e625622cc1a4224932ec4648602a49


**Additional context**
Revised version of the CX Chains Tutorial wiki was posted to the CXLabs Telegram group here:
https://t.me/CXLabs/856

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -11,8 +11,10 @@
 * Libraries
   * ...
 * Fixed issues
+  * #357: Error running cx in blockchain broadcast mode.
   * #360: Panic when package keyword is misspelled
   * #373: Error in address used to generate a CSRF token. Port was 6001 instead of 6421.
+  * #388: Array fail on cx --blockchain.
   * #389: CX chains errors with big programs.
 * Documentation
   * New file `documentation/BLOCKCHAIN-OVERVIEW.md` which describes the processes and modules involved in CX chains.

